docs/source/conf.py

```diff
@@ -23,7 +30,7 @@
 author = "Alan Cruickshank"
 
 # The full version, including alpha/beta/rc tags
-release = "0.3.6"
+release = stable_version

```

Comment:
Found that we were behind on versions in the docs, I added this "stable_version" @alanmcruickshank.

I'm using that in the "example" of a pre-commit configuration. Didn't want maintainers to have to think about bumping the version in multiple places

Reply:
I think that merging this will result in the live docs stable version becoming an alias for 0.4.0 which isn't released yet. I could be wrong though - @alanmcruickshank ?

Reply:
No, you're right! I should change that back for now

Reply:
@dmateusp: Did you mean to change this back before merging? It looks like it still says `stable_version` rather than `0.3.6`.

Reply:
@barrywhart https://github.com/sqlfluff/sqlfluff/pull/576/files#diff-3a64d1a0f41d37628ed69b17636e138abf6b56b19175f9d80cce28df4ccbbd42R4

Reply:
Got it, thanks!